src/validation.cpp

```diff
@@ -3986,15 +3986,15 @@ VerifyDBResult CVerifyDB::VerifyDB(
     BlockValidationState state;
```

Comment:
nit in a1c83166f33d44a0909d8789d3a4bc8b2539842e: Since this commit is a fixup (https://github.com/bitcoin/bitcoin/pull/25574#discussion_r985851229) to the first commit, it can be re-ordered right after the first commit, if you re-touch.

Reply:
done